build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -58,11 +58,17 @@
 @nodesEmitStreamBatchSizeMin = 50
 
 # configmap settings related fbit config
+@enableFbitInternalMetrics = false
 @fbitFlushIntervalSecs = 0
 @fbitTailBufferChunkSizeMBs = 0
 @fbitTailBufferMaxSizeMBs = 0
 @fbitTailMemBufLimitMBs = 0
 @fbitTailIgnoreOlder = ""
+@storageTotalLimitSizeMB = 200
+@outputForwardWorkers = 10
+# retries infinetly until it succeeds
+@outputForwardRetryLimit = "no_limits"
```

Comment:
> Is it safe to have infinite retries by default?

The core reason for the infinite retry to avoid data loss and tested, and should it fine. If required, customer can configure retry limit through config map.